src/modules/paymentwall/views/templates/hook/payment_infos.tpl

```diff
@@ -0,0 +1,14 @@
+<form action="#" method="post">
```

Comment:
`form` tag is not necessary at here. We don't have any `input` tag this template.
So, you can remove it.